wallet/lib/pages/purchase_item/purchase_item_screen.dart

```diff
@@ -198,6 +198,7 @@ class _OwnerBottomDrawerState extends State<OwnerBottomDrawer> {
   bool liked = false;
   bool collapsed = true;
   bool isExpanded = false;
+  bool balancesFetchResult = true;
```

Comment:
this should be in the method not outside.

Reply:
Alright